.github/workflows/publish.yaml

```diff
@@ -33,3 +35,9 @@ jobs:
 
       - name: Build and publish image
         run: ./scripts/build-image.sh
+
+      - name: Commit new contract metadata
+        run: |
+          git add js-library/src/contracts/
+          git commit -m "release: update contract metadata"
```

Comment:
`release` is not a valid conventional commit type. change it to either `chore` or `build`